src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -141,16 +142,33 @@ class ReferralService(
 
     /*
      * This is a temporary solution until a robust asynchronous link is created between Interventions
-     * and Delius. Once the asynchronous link is implemented, this feature can be turned off, and instead
-     * Delius will be notified via the normal asynchronous route
+     * and Delius/ARN. Once the asynchronous link is implemented, this feature can be turned off, and instead
+     * Delius/ARN will be notified via the normal asynchronous route
      */
+    submitAdditionalRiskInformation(referral, user)
     communityAPIReferralService.send(referral)
 
     val sentReferral = referralRepository.save(referral)
     eventPublisher.referralSentEvent(sentReferral)
     return sentReferral
   }
 
+  private fun submitAdditionalRiskInformation(referral: Referral, user: AuthUser) {
+    val riskId = referral.additionalNeedsInformation?.let { riskInformation ->
+      assessRisksAndNeedsService.createSupplementaryRisk(
+        referral.id,
+        referral.serviceUserCRN,
+        user.id,
+        OffsetDateTime.now(), // fixme: this should be the timestamp which we store when additionalRiskInformation is set
```

Comment:
You don't need to send us a timestamp here, if you don't we default it to LocalDateTime.now()

Reply:
we have a concept of this information potentially being somewhat outdated between the user inputing the risk data and submitting the form. sending the timestamp they actually entered the risk data seems like a good idea here. (as per the comment, that's what will eventually happen)